rosidl_generator_py/rosidl_generator_py/__init__.py

```diff
@@ -65,14 +72,20 @@ def generate_py(generator_arguments_file):
 
         module_name = convert_camel_case_to_lower_case_underscore(type_name)
         modules[subfolder].append((module_name, type_name))
-        for template_file, generated_filename in mapping.items():
-            data = {'package_name': args['package_name'], 'spec': spec}
-            data.update(functions)
-            generated_file = os.path.join(
-                args['output_dir'], subfolder, generated_filename % module_name)
-            expand_template(
-                template_file, data, generated_file,
-                minimum_timestamp=latest_target_timestamp)
+        for template_file, generated_filenames in mapping.items():
+            for generated_filename in generated_filenames:
+                data = {
+                    'module_name': module_name, 'package_name': args['package_name'],
+                    'spec': spec, 'subfolder': subfolder,
```

Comment:
Please wrap the lines after each comma since it makes it very hard to see what keys are actually in the dict otherwise.